### PR TITLE
#682 Fix proposal pdf layout

### PR DESCRIPTION
**Pull request by AlexBelov**
Fixing pdf layout  
Old:  
* https://dl.dropboxusercontent.com/u/36906193/9%20sem/Projects/airesis/old.pdf  

New:  
* https://dl.dropboxusercontent.com/u/36906193/9%20sem/Projects/airesis/bike.pdf
* https://dl.dropboxusercontent.com/u/36906193/9%20sem/Projects/airesis/dragon.pdf

### CODE REVIEW

**Comment by leoniglio** [comment]
Sembra un buon lavoro!

File: app/models/paragraph.rb [hunk 1]
```diff
@@ -24,4 +24,8 @@ def content=(content)
     ed_content = '<p></p>' if ed_content.to_s == ''
     write_attribute(:content, ed_content)
   end
+
+  def empty?
+    content == "<p></p>"
```
Comment (coorasse) [inline app/models/paragraph.rb]: :+1:

File: app/views/proposals/_rank_left_panel.html.erb [hunk 2]
```diff
@@ -26,21 +26,19 @@
...
-    <% if @proposal.voted? %>
-        <% if params[:action] == 'show' %>
-            <% if @proposal.vote %>
-                <!--todo wrong url-->
-                <%= link_to vote_results_proposal_url(@proposal), remote: true, class: 'btn disable results-button' do %>
-                    <%= t('pages.proposals.show.votation_results') %>
-                <% end %>
-                <br/><br/>
+    <% if params[:action] == 'show' %>
+        <% if @proposal.vote %>
```
Comment (coorasse) [inline app/views/proposals/_rank_left_panel.html.erb]: voted?

File: app/assets/stylesheets/pdf/proposal.css.scss [hunk 3]
```diff
@@ -0,0 +1,129 @@
+.proposalMainContainer .proposal_content {
```
Comment (coorasse) [inline app/assets/stylesheets/pdf/proposal.css.scss]: try not to use camelCase css.

File: app/assets/stylesheets/pdf/proposal.css.scss [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    font: 14px, arial;
+    letter-spacing: 0.6px;
+    word-wrap: normal;
+}
+
+.wicked_center_panel {
```
Comment (coorasse) [inline app/assets/stylesheets/pdf/proposal.css.scss]: underscores as well, prefer `-`

File: config/initializers/wkhtmltopdf.rb [hunk 5]
```diff
@@ -1 +1 @@
-WickedPdf.config = { exe_path: '/home/coorasse/AiresisWorkspace/wkhtmltopdf9' }
+WickedPdf.config = { exe_path: `which wkhtmltopdf`.strip }
```
Comment (coorasse) [inline config/initializers/wkhtmltopdf.rb]: aha. yeah. cool :)

File: config/locales/main/en-EU.yml [hunk 6]
```diff
@@ -482,6 +482,10 @@ en-EU:
         you_can_choose_vote_dates: "This proposal has been waiting for %{count} days for a voting date, so you are now allowed to choose one."
         no_contributes_yet: "No contributions here yet"
         start_votation_proposal_confirmation: "Are you sure you want to start immediatly the votation?"
+        debate_begin: "Begin of debate"
+        debate_end: "End of debate"
+        vote_begin: "Begin of voting"
+        vote_end: "End of voting"
```
Comment (coorasse) [inline config/locales/main/en-EU.yml]: aren't already somewhere these strings?

File: app/views/proposals/_rank_left_panel.html.erb (hunk 2 above)
Comment (AlexBelov) [inline app/views/proposals/_rank_left_panel.html.erb]: this allow us to export pdf in voting phase

File: app/models/paragraph.rb (hunk 1 above)
Comment (AlexBelov) [inline app/models/paragraph.rb]: It makes sense...

File: config/locales/main/en-EU.yml (hunk 6 above)
Comment (AlexBelov) [inline config/locales/main/en-EU.yml]: Yup, I've just found 
vote_starts_at: "Vote starts at"
vote_ends_at: "Vote ends at"
in activerecord locales. But can't find something similar for debate phase - so added it
debate_starts_at: "Debate starts at"
debate_ends_at: "Debate ends at"

**Pull request merged**